cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -53,7 +55,20 @@ object CqlParser extends JavaTokenParsers
    */
   def constant: Parser[Constant] = {
     import Constants._
-    (string | uuid | number | boolean) ^^ Constant // | hex // TODO
```

Comment:
Also a comment in the tests to describe that NaN is case sensitive while Null is not.